src/gui/widgets/backupwidget.cpp

```diff
@@ -89,21 +89,29 @@ void BackupWidget::fillTableWithSampleData()
         filenames.append(item);
     }
 
-    // Second column, would be created by reading the details from each file and creating the description string
-    QList<QStandardItem *> descriptions;
+    // Get summary of each db file and populate lists (columns) of data
+    QList<QStandardItem *> total_flights, total_tails, total_pilots, max_doft, total_time;
     for (const auto &entry : entries) {
-        QStringList summary = aDB->databaseSummary(entry);
-        auto item = new QStandardItem(summary.join(", "));
-        descriptions.prepend(item);  // Make a O(1) op here and deal with sorting later
+        QMap<QString, QString> summary = aDB->databaseSummary(backup_dir.absoluteFilePath(entry));
+        total_flights.prepend(new QStandardItem(summary["total_flights"]));
```

Comment:
Are you using prepend over append for some efficiency reason or is it just personal preference? (I don't mind, just curious)

Reply:
My comment is my reason. Append is an O(n) operation where as prepend is O(1) and i dont see any reason to waste n complexity time. The thing is though it may reverse the sort order that way so we should check it out.
We should check what entryList returns (ascending or descending) and actually put the opossite of what we want.
All of this could have bern avoided if the tableview accepted vector of items but for some reason it doesnt.

Reply:
Maybe we can save us the hassle of dealing with reversing - I think Qt's implementation of QList differs from the standard behavior you are expecting - According to the [docs](https://doc.qt.io/qt-5/containers.html#algorithmic-complexity), append and prepend are both O(1).

![qlist](https://user-images.githubusercontent.com/32618887/110210670-025eed80-7e93-11eb-82ab-3defa4258e86.png)

The docs for [QList](https://doc.qt.io/qt-5/qlist.html#append) also have an identical comment for append and prepend 
> If this list is not shared, this operation [append or prepend] is typically very fast (amortized constant time), because QList preallocates extra space on both sides of its internal buffer to allow for fast growth at both ends of the list.

Reply:
oooh cmon... QList is a variation of an array not std::list. QLinkedList is literally a list...
Then no reason not to use append